packages/carbon-web-components/src/components/ui-shell/ui-shell-story.ts

```diff
@@ -266,36 +286,138 @@ export const sideNavWithIcons = (args) => {
   return result;
 };
 
-sideNavWithIcons.storyName = 'Side nav with icons';
+FixedSideNavIcons.storyName = 'Fixed SideNav w/ Icons';
+
+export const HeaderBase = () => {
+  return html` <style>
+      ${styles}
+    </style>
+    <cds-header aria-label="IBM Platform Name">
+      <cds-header-name href="javascript:void 0" prefix="IBM">
+        [Platform]
+      </cds-header-name>
+    </cds-header>`;
+};
 
-sideNavWithIcons.parameters = {
-  knobs: sideNav.parameters.knobs,
+export const HeaderBaseWActions = () => {
+  return html` <style>
+      ${styles}
+    </style>
+    <cds-header aria-label="IBM Platform Name">
+      <cds-header-name href="javascript:void 0" prefix="IBM">
+        [Platform]
+      </cds-header-name>
+      <div class="${prefix}--header__global">
+        <cds-header-global-action aria-label="Search" tooltip-text="Search">
+          ${Search20({ slot: 'icon' })}
+        </cds-header-global-action>
+        <cds-header-global-action
+          aria-label="Notification"
+          tooltip-text="Notification">
+          ${Notification20({ slot: 'icon' })}
+        </cds-header-global-action>
+        <cds-header-global-action
+          aria-label="App Switcher"
+          tooltip-text="App Switcher"
+          tooltip-alignment="end">
```

Comment:
```suggestion
          tooltip-alignment="right">
```

Using `right` will apply the proper styles to the tooltip, you can search and replace this one for the rest in this file

Reply:
thanks!